frontend/src/pages/modelServing/screens/metrics/bias/BiasConfigurationPage/BiasConfigurationModal/MetricTypeField.tsx

```diff
@@ -33,13 +35,9 @@ const MetricTypeField: React.FC<MetricTypeFieldProps> = ({ fieldId, value, onCha
         selections={value}
         menuAppendTo="parent"
       >
-        {Object.keys(BiasMetricType).map((type) => (
-          <SelectOption
-            key={type}
-            value={type}
-            description={METRIC_TYPE_DESCRIPTION[type as keyof typeof BiasMetricType]}
-          >
-            {METRIC_TYPE_DISPLAY_NAME[type as keyof typeof BiasMetricType]}
+        {enumIterator(BiasMetricType).map(([, type]) => (
+          <SelectOption key={type} value={type} description={METRIC_TYPE_DESCRIPTION[type]}>
+            {METRIC_TYPE_DISPLAY_NAME[type]}
```

Comment:
Shouldn't this use the first entry in the iterator since the original code used `Object.keys`
```suggestion
        {enumIterator(BiasMetricType).map(([type]) => (
          <SelectOption key={type} value={type} description={METRIC_TYPE_DESCRIPTION[type]}>
            {METRIC_TYPE_DISPLAY_NAME[type]}
```